tools/doc/html.js

```diff
@@ -165,10 +175,32 @@ function parseLists(input) {
 }
 
 
+// Syscalls which appear in the docs, but which only exist in BSD / OSX
+var BSD_ONLY_SYSCALLS = new Set(['lchmod']);
+
+// Handle references to man pages, eg "open(2)" or "lchmod(2)"
+// Returns modified text, with such refs replace with HTML links, for example
+// '<a href="http://man7.org/linux/man-pages/man2/open.2.html">open(2)</a>'
+function linkManPages(text) {
+  return text.replace(/ ([a-z]+)\((\d)\)/gm, function(match, name, number) {
```

Comment:
NVM. Here's a place where it fails

```
doc/api/modules.markdown:var mySquare = square(2);
```

I'll change it to require special markup to generate the link, so we don't get false positives.